### PR TITLE
Division + Refactoring

### PR DESCRIPTION
**Pull request by JuhoKon**
Added a division functionality and refactored the main file.

### CODE REVIEW

File: functions/math/math.js [hunk 1]
```diff
@@ -4,4 +4,7 @@ function sum(a, b) {
 function substraction(a, b) {
   return a - b;
 }
-module.exports = { sum, substraction };
+function division(a, b) {
+  return a / b;
```
Comment (JuhoKon) [inline functions/math/math.js:8]: Division by zero is not handled properly!

File: functions/math/math.test.js [hunk 2]
```diff
@@ -7,3 +7,6 @@ test("adds 5 + 6 to equal 3", () => {
 test("substraction 5 - 6 to equal -1", () => {
   expect(substraction(5, 6)).toBe(-1);
 });
+test("division 10 / 5 to equal 2", () => {
```
Comment (JuhoKon) [inline functions/math/math.test.js:10]: Add a test case for division by zero

File: index.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
-const { sum } = require("./functions/math/math");
+const { sum, substraction, division } = require("./functions/math/math");
 
+function handleoperations(a, b) {
```
Comment (JuhoKon) [inline index.js:3]: Please follow camelCase naming convention for methods!
